.directory

```diff
@@ -0,0 +1,6 @@
+[Dolphin]
+Timestamp=2016,12,16,19,27,3
+Version=3
+
+[Settings]
+HiddenFilesShown=true
```

Comment:
???? Do not submit this!

Always run `git status`
and then `git add FILEPATH`
Do not run `git add *` or `git add .`